src/ontology/hp.Makefile

```diff
@@ -426,3 +426,20 @@ help:
 	@echo "$$data"
 	echo "Migrating EQs from MP to HP:"
 	echo "make ADOPT_EQS_MAPPING_URL=SOMEURL migrate_eqs_to_edit"
+
+.PHONY: diff
+diff: diff-table diff-kgcl diff-yaml
+
+tmp/hp-released.obo:
+	wget http://purl.obolibrary.org/obo/hp.obo -O $@ && cp ../../hp-full.obo tmp/hp-full.obo
+
+diff-table: tmp/hp-released.obo tmp/hp-full.obo
+	runoak -i simpleobo:tmp/hp-full.obo diff -X simpleobo:tmp/hp-released.obo \
+	-o difference_table.tsv --output-type csv --statistics --group-by-property oio:hasOBONamespace
```

Comment:
$(REPORTDIR)/difference_table.tsv - only use `.PHONY` goals as shortcuts.

Reply:
Can the output be markdown?

Reply:
`--output-format md` throws an error at the moment. Regardless it does not (I think) give the kind of output you expect as mentioned below like mondo.

Reply:
Ok, this smells like we need to focus our effort for now on oak!